apps/fluent-tester/src/E2E/Menu/pages/MenuPageObject.ts

```diff
@@ -78,6 +74,10 @@ class MenuPageObject extends BasePage {
     return await this._primaryComponent;
   }
 
+  async resetTest() {
+    await (await this._defocusButton).click();
```

Comment:
Does this actually close the Menu? I just tried on the tester app and it didn't. Can you confirm?

Reply:
it should light dismiss off a click (testable :D) but also `escape` is a surefire menu dismissal

Reply:
What is a "light" dismiss?

Reply:
@lawrencewin fyi for context; `light dismiss UI` is transient UI such as callouts and menus that have less screen presence/permanence than dialogs (transient UI that requires explicit user dismissal).  The most common action is a tap/click away from the light-dismiss UI, but ranges in behavior between features and apps. [Our Callout on Win32 currently offers](https://github.com/microsoft/fluentui-react-native/blob/6dea077f3b2fc93fc533128ecbc58d00cc9644ed/packages/components/Callout/src/Callout.types.ts#L29) `preventDismissOnKeyDown` and `preventDismissOnClickOutside` to disable these common light-dismiss interactions, which could be used in UI such as an @ mention picker or a teaching UI bubble (respectively) to allow them to stay on-screen longer for intentional behavior design or to avoid unintended user dismissals.

Reply:
> Does this actually close the Menu? I just tried on the tester app and it didn't. Can you confirm?

Works on my machine, though I did add an escape press on the menu trigger button. I also keep the click on the defocus button where only pressing escape somehow breaks the e2etests and resetting focus somehow fixes that.